src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -216,6 +216,12 @@ def load_command_table(self, _):
         g.custom_command('add', 'add_hc')
         g.custom_command('remove', 'remove_hc')
 
+    with self.command_group('webapp private-endpoint-connection', is_preview=True) as g:
```

Comment:
talking with Network team, these commands should be in `az network` command group. Could you follow this [doc](https://github.com/Azure/azure-cli/blob/dev/doc/private_endpoint_connection_command_guideline.md)